### ISSUE TITLE
✨ FR: Multiple projects in one

### ISSUE THREAD

**Issue by sschneider-ihre-pvs** [SpectralOps/teller#59]
Use case, if you have a frontend and a backend project and the frontend one has the backend url/port as env var and the backend has an env var vor its port also both would need a teller file and the teller file needs to be in the project folder.
Instead, an option for having a central teller file for multiple possibly overlapping project that don't have to be checked in at all and can reside in the home directory would be nice.

**Comment by jondot** [comment]
Got it.
So to re-iterate, a single config file, that outline multiple projects.
Lets say we have a monorepo:

```
backend/
frontend/
some.json
other.json
teller.yml <--- this is a single config
```

And our yaml file will map:

```yaml

projects:
  backend:
    [...]
  frontend:
   [...]
```
and somehow, let Teller via CLI pick which project you want.

Is that accurate?

**Comment by sschneider-ihre-pvs** [comment]
yes pretty much, but maybe also that teller looks for a .tellery.yaml in the home folder also if it doesn't find one and the projects need to have some kind of path set to it

**Comment by jondot** [comment]
And do you foresee some settings/secrets that are common to both backend and frontend as well?

**Comment by sschneider-ihre-pvs** [comment]
port settings for example. backend running on port and frontend accessing port of said backend
